puppet/modules/quickstack/manifests/pacemaker/galera.pp

```diff
@@ -20,16 +20,6 @@
   if (str2bool_i(map_params('include_mysql'))) {
     $galera_vip = map_params("db_vip")
 
-    # TODO: extract this into a helper function
-    if ($::pcs_setup_galera ==  undef or
-        !str2bool_i("$::pcs_setup_galera")) {
-      $_enabled = true
-      $_ensure = 'running'
-    } else {
-      $_enabled = false
-      $_ensure = undef
-    }
-
```

Comment:
I just updated this PR with the above deletions.   $_enabled and $_ensure have not been used in this class for awhile.